usaddress/__init__.py

```diff
@@ -61,72 +61,81 @@
                   'northeast', 'northwest', 'southeast', 'southwest'])
 
 STREET_NAMES = {
-    'allee', 'alley', 'ally', 'aly', 'anex', 'annex', 'annx', 'anx', 'arc',
```

Comment:
I apologize for this. I figured maintaining an alphabetical list was preferable without thinking about the diff.  This is the correct list